daemon/cmd/daemon_main.go

```diff
@@ -1590,6 +1591,7 @@ type daemonParams struct {
 	EndpointManager      endpointmanager.EndpointManager
 	CertManager          certificatemanager.CertificateManager
 	SecretManager        certificatemanager.SecretManager
+	SignalManager        signal.SignalManager
```

Comment:
`SignalManager` isn't used. it should be enough to have the signal cell itself registered in the daemon cell and have an explicit `invoke` in the cell itself (which both should be already the case).

or do we have some special initialization order requirements that i'm missing?

Reply:
Not that I know of, removed this :-)